sdk/src/tokenFaucet.ts

```diff
@@ -19,27 +17,33 @@ import {
 import { BN } from '.';
 import tokenFaucet from './idl/token_faucet.json';
 import { IWallet } from './types';
+import { BankrunContextWrapper } from './bankrunConnection';
 
 export class TokenFaucet {
-	connection: Connection;
+	context: BankrunContextWrapper;
```

Comment:
i think the devnet ui uses this and this might cause that to break, can we keep this as a connection?